searx/shared/shared_abstract.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+class SharedDict:
```

Comment:
Why don't you use Abstract Base Class here?

Reply:
Fixed in this commit https://github.com/searx/searx/pull/2419/commits/eca200404d195fcfaf29a8b30f3cc7d2c7e4c021